eslint.config.js

```diff
@@ -1,25 +1,32 @@
 const ts = require('typescript-eslint');
 const js = require('@eslint/js');
 const prettier = require('eslint-config-prettier');
-const cypress = require('eslint-plugin-cypress');
+const cypress = require('eslint-plugin-cypress/flat');
+const jest = require('eslint-plugin-jest');
 
 module.exports = ts.config(
   js.configs.recommended,
   ...ts.configs.recommended,
   prettier,
-  {
-    plugins: { cypress: cypress },
-  },
   {
     ignores: ['lib/**'],
   },
   {
-    files: ['**/*.js', '**/*.ts'],
+    files: ['**/*.*js', '**/*.*ts'],
     rules: {
       '@typescript-eslint/no-var-requires': 'off',
+      '@typescript-eslint/no-explicit-any': 'warn',
       'no-undef': 'warn',
     },
   },
+  {
+    files: ['tests/**/*.*js', 'tests/**/*.*ts'],
```

Comment:
The current configuration would mistake `.cy.js` for jest. But those are cypress.
```suggestion
    files: ['tests/**/*.spec.js', 'tests/**/*.spec.ts'],
```